src/applications/travel-pay/constants.js

```diff
@@ -0,0 +1,129 @@
+export const STATUS_GROUPINGS = [
+  {
+    name: 'Saved or Incomplete',
+    description:
+      'These are claims that the Beneficiary Travel office cannot process. Either you have not submitted the claim, or you submitted a claim without required information.',
+    includes: ['Incomplete', 'Saved'],
+  },
+  {
+    name: 'Under VA Review',
+    description:
+      'These claims require action from VA. If VA needs more information from you, your Travel Clerk will contact you.',
+    includes: [
+      'In Process',
+      'Claim Submitted',
+      'In Manual Review',
+      'On Hold',
+      'Appealed',
+    ],
+  },
+  {
+    name: 'Closed',
+    description:
+      'The Beneficiary Travel office finished their review of your claim and closed it. In some situations you can appeal Beneficiary Travel department’s decision and re-open a claim. If you have questions about why your claim has one of the following statuses, contact your local VAMC and ask for the Beneficiary Travel department.',
+    includes: [
+      'Partial Payment',
+      'Denied',
+      'Approved for Payment',
+      'Submitted for Payment',
+      'Fiscal Rescinded',
+      'Claim Paid',
+      'Payment Canceled',
```

Comment:
This overall data structure looks really nice, great job. One suggestion to make the rendering code more seamless is to turn the statuses in the `includes` array from logical links to physical links:

```javascript
{
  // ...
  includes: [
    PARTIAL_PAYMENT,
    DENIED,
    // ...
  ]
}

PARTIA_PAYMENT = {
  name: 'Partial Payment',
  description: 'paid AND unpaid',
  reasons: null,
};

```

This means the individual status objects live outside the `STATUSES` array for reference by the `STATUS_GROUPINGS` object.

Reply:
So you're thinking each status object is it's own `const` rather than an array of all the statuses?

Reply:
Yeah. We could still have an array of `STATUSES` if it's needed, but I'm not sure the need is there. Please push back if you're not a fan!

Reply:
🤔 My knee jerk reaction is that would be importing each individual status object on the page vs. just the status array and the grouping array, which might be overkill, but maybe not? 

Oooo.... Maybe the status array becomes an object, then we import just that object, and rather than mapping over it we can say `STATUSES[PARTIAL_PAY]` or something....

Reply:
RE importing each status - I was thinking you'd reference them in the includes array on the status groupings object, and then just be able to import that.

Reply:
Tried, didn't work. ☹️